backend/src/test/resolvers.spec.js

```diff
@@ -0,0 +1,590 @@
+import { createTestClient } from "apollo-server-testing";
+import { gql } from "apollo-server";
+import Server from "../server";
+import { GraphQLError } from "graphql";
+import { context } from "../context"
+
+jest.mock("../graphCms/schema");
+
+let reqMock;
+let resMock;
+let query;
+let mutate;
+let contextMock = () => context({ req: reqMock, res: resMock });
+
+beforeEach(async () => {
+  const server = await Server({ context: () => contextMock });
```

Comment:
That's why the tests are taking so long. You're re-initializing the entire server for ever test case. Because the only data that is being changed in tests is `dataSources`, `context` and your mock `GraphCmsSchema`, you don't have to re-initialize the whole server, just the parts that change.